test/json-schema-files/winservices-schema.json

```diff
@@ -14,7 +14,7 @@
           "type": "string"
         },
         "version": {
-          "pattern": "^[0-9]+.[0-9]+.[0-9]+$",
+          "pattern": "^v[0-9]+.[0-9]+.[0-9]+(-beta)?$",
```

Comment:
the test failed here, i guess this should be here https://regex101.com/r/Ly7O1x/3/ but for me is too complex , we just need to check if this is not empty or default , i think `^v[0-9]+\.[0-9]+\.[0-9]+` should work